### PR TITLE
electron-37: fixed issues w.r.t. installer pane not capturing the pod…

### PR DESCRIPTION
**Pull request by VishwasShashidhar**
a new PR to correct merge issues as per PR #111

@lneir Please check this now. It has clean code.

### CODE REVIEW

File: installer/mac/postinstall.sh [hunk 1]
```diff
@@ -16,5 +16,33 @@ sed -i "" -E "s#\"url\" ?: ?\".*\"#\"url\"\: \"$pod_url\"#g" $newPath
 sed -i "" -E "s#\"minimizeOnClose\" ?: ?([Tt][Rr][Uu][Ee]|[Ff][Aa][Ll][Ss][Ee])#\"minimizeOnClose\":\ $minimize_on_close#g" $newPath
 sed -i "" -E "s#\"launchOnStartup\" ?: ?([Tt][Rr][Uu][Ee]|[Ff][Aa][Ll][Ss][Ee])#\"launchOnStartup\":\ $launch_on_startup#g" $newPath
 
+## Add app to login items
+if [ $launch_on_startup == true ]; then
```
Comment (lneir) [inline installer/mac/postinstall.sh]: looks like in this PR you are also adding auto-launch setup?  and also fixing pod url issue?  generally we should put separate issues in different PRs.

**Comment by VishwasShashidhar** [comment]
The idea was to bundle this logic with this branch because we were changing something with regards to the installer. 

But if it has to be done separately, will do that.

Sent from my iPhone

On 29-May-2017, at 8:44 PM, Lynn <notifications@github.com> wrote:

@lneir commented on this pull request.

In installer/mac/postinstall.sh:

> @@ -16,5 +16,33 @@ sed -i "" -E "s#\"url\" ?: ?\".*\"#\"url\"\: \"$pod_url\"#g" $newPath
 sed -i "" -E "s#\"minimizeOnClose\" ?: ?([Tt][Rr][Uu][Ee]|[Ff][Aa][Ll][Ss][Ee])#\"minimizeOnClose\":\ $minimize_on_close#g" $newPath
 sed -i "" -E "s#\"launchOnStartup\" ?: ?([Tt][Rr][Uu][Ee]|[Ff][Aa][Ll][Ss][Ee])#\"launchOnStartup\":\ $launch_on_startup#g" $newPath
 
+## Add app to login items
+if [ $launch_on_startup == true ]; then
looks like in this PR you are also adding auto-launch setup? and also fixing pod url issue? generally we should put separate issues in different PRs.

File: installer/mac/postinstall.sh (hunk 1 above)
Comment (lneir) [inline installer/mac/postinstall.sh]: yes in future, let's make sure PR is working on one thing.
